README.md

```diff
@@ -10,13 +10,15 @@ This is the rendering application for the Book a secure move service.
 Book a secure move is part of the HMPPS Prisoner Escort and Custody
 Service (PECS) programme.
 
-## Dependencies
+## Quick Start
+
+### Major Dependencies
 
 - [Node.js](https://nodejs.org/en/) (>= 14)
```

Comment:
Is it possible that we could piggyback of the release process and update these values too?

Reply:
Interesting idea - I've created #846 to capture it